src/parquet/column/properties.h

```diff
@@ -121,9 +128,16 @@ class WriterProperties {
       return this;
     }
 
+    Builder* set_compression(const std::shared_ptr<schema::ColumnPath>& path,
```

Comment:
Can you re-add the version that sets a default for all columns for which not an explicit codec is set?
Also please remove the `set_` prefix to. Be in line with the other methods. 

We might also want to have these methods with a string as the first argument as this will be easier to use for the end-user.